kano_settings/system/advanced.py

```diff
@@ -430,7 +430,10 @@ def redirect_traffic_to_localhost():
 
 
 def launch_sentry_server(filename):
-    subprocess.Popen(["sentry -c {}".format(filename)], shell=True)
+    subprocess.Popen(
```

Comment:
@convolu That is a good question and the only reason that I have is that presumably it wasn't used originally for some reason.